### PR TITLE
Try: Update noon & typography preset 1.

### LINKED ISSUE #547: Noon: List of elements to adjust
While testing the **Noon variation and its typography preset**, I noticed some elements could be improved:

Image
Image

Here are the changes that can improve the look of the variation & typeset:

- Reset Post Author styles
- Reset Post Date styles
- Post Terms: reset font and font size
- Reset Titles typography styles
- Text: set letter spacing `-0.01em` and line height `1,6`
- All Headings: set letter spacing `-0.02em` , line height `1,02`
- H3: set to L font size
- H5: remove custom letter spacing
- Text preset size XL: min `2rem`, max `2.8rem`
- Text preset size XL: min `1.8rem`, max `2.2rem`
- Text preset size L: min `1.6rem`, max `1.8rem`
- Text preset size Medium: min `1rem`, max `1.2 rem`
- Quote: remove line height (it should be the same as text)

How it's expected to look with the changes applied.

Image
Image
Image
Image
Image

cc: @juanfra @carolinan @richtabor

### PR DESCRIPTION
**Pull request by juanfra**
**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/547

Updating the style variation and typography preset according to the request [here](https://github.com/WordPress/twentytwentyfive/issues/547)

**Screenshots**

- N/A

**Testing Instructions**

1. Go to the site editor
2. Pick the "Beiruti & Literata" typographic preset and confirm the styles match the ones listed in the issue.

---

1. Go to the site editor
2. Pick the "Noon" style variation and confirm the styles match the ones listed in the issue.

### CODE REVIEW

File: styles/typography/typography-preset-1.json [hunk 1]
```diff
@@ -118,7 +93,9 @@
 			},
 			"core/quote": {
 				"typography": {
-					"fontWeight": "300"
+					"fontSize": "var:preset|font-size|large",
+					"fontWeight": "300",
```
Comment (carolinan) [inline styles/typography/typography-preset-1.json]: This font weight and font size are is the same as in theme.json so it does not need to be duplicated.
The line height is the same as on the body so it does not need to be duplicated.

File: styles/typography/typography-preset-1.json [hunk 2]
```diff
@@ -138,7 +115,7 @@
 		"elements": {
 			"h3": {
 				"typography": {
-					"fontSize": "1.8rem"
+					"fontSize": "var:preset|font-size|large"
```
Comment (carolinan) [inline styles/typography/typography-preset-1.json]: This is the same size as in theme.json so it does not need to be duplicated.

**Comment by juanfra** [comment]
Thank you Carolina!

**Comment by carolinan** [comment]
- [x] Reset Post Author styles
- [x] Reset Post Date styles
- [x] Post Terms: reset font and font size
- [x] Reset Post Titles typography styles
- [x] Text: set letter spacing -0.01em and line height 1,6
- [x] All Headings: set letter spacing -0.02em , line height 1,02
- [inherits] H3: set to L font size
- [x] H5: remove custom letter spacing

- [x] Assumed XXL     Text preset size XL: min 2rem, max 2.8rem
- [x] Text preset size XL: min 1.8rem, max 2.2rem
- [x] Text preset size L: min 1.6rem, max 1.8rem
- [x] Text preset size Medium: min 1rem, max 1.2 rem

- [ ] Quote: remove line height (it should be the same as text)

**Comment by beafialho** [comment]
Thanks @juanfra!

The only thing I'm seeing differently than expected is the quote:

| What I'm seeing | What it's expected to look like |
| ----- | ---- |
| Captura de ecrã 2024-10-14, às 11 38 41 |    375330552-8c4c7a9e-0b46-4409-8d3f-3cf150b0294d   |

**Comment by beafialho** [comment]
Thank you @juanfra, the quote is looking good now 👍

**Comment by juanfra** [comment]
Thank you! Merging this one.

**Pull request merged**